### ISSUE TITLE
Change fixed radiation variables into parameters.

### ISSUE THREAD

**Issue by damienpicard** [open-ideas/IDEAS#339]
The radiation calculations in `IDEAS.Buildings.Components.BaseClasses.ZoneLwGainDistribution` are using variable with fixed values during the simulation.

```
protected 
  Real[nSurf] areaAbsDifSol=area .* epsSw "longwave emissivity weighted areas";
  Real areaAbsDifTotSol=sum(areaAbsDifSol) 
    "sum of longwave emissivity weighted areas";
  Real[nSurf] areaAbsGain=area .* epsLw "shortwave emissivity weighted areas";
  Real areaAbsTotGain=sum(areaAbsGain) 
    "sum of shortwave emissivity weighted areas";

  Real[nSurf] weightFactorDir = 0.96 * area ./ (ones(nSurf)*sum(area)) 
    "weightfactor for received direct shortwave solar radiation";
  Real[nSurf] weightFactorDif = 0.96 * areaAbsDifSol ./ (ones(nSurf)*areaAbsDifTotSol) 
    "weightfactor for received direct shortwave solar radiation";
  Real[nSurf] weightFactorGain = areaAbsGain ./ (ones(nSurf)*areaAbsTotGain) 
    "weightfactor for received direct shortwave solar radiation";
```

Would it be possible to make them as parameter with `fixed` = false and defined their values in initial algorithm? That would reduce the number of equations.

**Comment by rubenbaetens** [comment]
@damienpicard they were (once) defined as a variable as we couldn't propagate paramaters through input/output-ports, while indeed they may be considered as a parameter.

**Comment by damienpicard** [comment]
Actually, I wonder if the inputs couldn't be propagated as parameters using propbus?

**Comment by Mathadon** [comment]
@rubenbaetens can you have a look at this?

**Comment by damienpicard** [comment]
see pull request #357

**Comment by damienpicard** [comment]
This has been solved in #357
